gcip/addons/aws/helper.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+import boto3
+
+session = boto3.Session()
+sts = session.client("sts")
```

Comment:
`session` is never used

```suggestion
sts = boto3.Session().client("sts")
```

Reply:
Accepted